modules/ROOT/pages/rtm-agent-proxy-config.adoc

```diff
@@ -57,6 +57,28 @@ This example configures the agent to work with a proxy server that requires auth
 amc_setup -H myToken myMuleServer -P acme.proxy.com 443 internalAdmin Ins1d3V0icePassword
 ----
 
+== Verify That the Proxy Server Does Not Modify the Agent Certificate 
+
+To ensure that your firewall or proxy does not intercept or modify the 
+Runtime Manager agent, run the following command:
+
+[source,console,linenums]
+----
+echo -e "GET / HTTP/1.0\r\n" | openssl s_client -connect mule-manager.anypoint.mulesoft.com:443 -ign_eof
+----
+
+The output of this command should include the following information:
+
+[source,console,linenums]
+----
+-----END CERTIFICATE-----
+subject=/C=US/O=Hybrid/OU=MuleSoft/CN=mule-manager.anypoint.mulesoft.com
```

Comment:
For `runtime-manager.anypoint.mulesoft.com`

the output would be

`
-----END CERTIFICATE-----
subject=/C=US/ST=ca/L=San Francisco/O=MuleSoft, LLC/OU=Mulesoft/CN=runtime-manager.anypoint.mulesoft.com
issuer=/C=US/O=DigiCert Inc/CN=DigiCert SHA2 Secure Server CA
---`